.flutter-plugins

```diff
@@ -0,0 +1,36 @@
+# This is a generated file; do not edit or check into version control.
+device_info_plus=/Users/nisainob/Development/flutter/.pub-cache/hosted/pub.dartlang.org/device_info_plus-2.2.0/
+device_info_plus_linux=/Users/nisainob/Development/flutter/.pub-cache/hosted/pub.dartlang.org/device_info_plus_linux-2.1.1/
```

Comment:
@nobnisai I think that these 2 files (`.flutter-plugins` and `.flutter-plugins-dependencies`) include your local environment information like `/Users/nisainob/Development/...`, so better to remove these files and add these file paths to `.gitignore`.